chore: prod build errors

not entirely sure what's causing this, prod build runs fine locally. could be because of AOT or the runtime: locally we're using a proper JDK, prod is just a JRE

docker logs show:
```
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    | Exception in thread "main" java.lang.ExceptionInInitializerError
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.Class.forName0(Native Method)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.Class.forName(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.classForName(RT.java:2229)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.classForName(RT.java:2238)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.loadClassForName(RT.java:2257)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.load(RT.java:469)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.load(RT.java:444)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load$fn__6933.invoke(core.clj:6189)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load.invokeStatic(core.clj:6188)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load.doInvoke(core.clj:6172)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.invoke(RestFn.java:411)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_one.invokeStatic(core.clj:5961)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_one.invoke(core.clj:5956)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_lib$fn__6875.invoke(core.clj:6003)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_lib.invokeStatic(core.clj:6002)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_lib.doInvoke(core.clj:5981)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.applyTo(RestFn.java:145)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$apply.invokeStatic(core.clj:669)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_libs.invokeStatic(core.clj:6044)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_libs.doInvoke(core.clj:6028)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.applyTo(RestFn.java:140)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$apply.invokeStatic(core.clj:669)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$require.invokeStatic(core.clj:6066)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$require.doInvoke(core.clj:6066)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.applyTo(RestFn.java:140)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$apply.invokeStatic(core.clj:667)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$serialized_require.invokeStatic(core.clj:6142)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$requiring_resolve.invokeStatic(core.clj:6151)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$requiring_resolve.invoke(core.clj:6145)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at orchard.java__init.load(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at orchard.java__init.<clinit>(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.Class.forName0(Native Method)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.Class.forName(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.classForName(RT.java:2229)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.classForName(RT.java:2238)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.loadClassForName(RT.java:2257)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.load(RT.java:469)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.load(RT.java:444)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load$fn__6933.invoke(core.clj:6189)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load.invokeStatic(core.clj:6188)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load.doInvoke(core.clj:6172)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.invoke(RestFn.java:411)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_one.invokeStatic(core.clj:5961)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_one.invoke(core.clj:5956)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_lib$fn__6875.invoke(core.clj:6003)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_lib.invokeStatic(core.clj:6002)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_lib.doInvoke(core.clj:5981)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.applyTo(RestFn.java:145)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$apply.invokeStatic(core.clj:669)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_libs.invokeStatic(core.clj:6044)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_libs.doInvoke(core.clj:6028)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.applyTo(RestFn.java:140)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$apply.invokeStatic(core.clj:669)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$require.invokeStatic(core.clj:6066)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at orchard.info$loading__6814__auto____35882.invoke(info.clj:1)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at orchard.info__init.load(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at orchard.info__init.<clinit>(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.Class.forName0(Native Method)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.Class.forName(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.classForName(RT.java:2229)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.classForName(RT.java:2238)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.loadClassForName(RT.java:2257)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.load(RT.java:469)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.load(RT.java:444)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load$fn__6933.invoke(core.clj:6189)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load.invokeStatic(core.clj:6188)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load.doInvoke(core.clj:6172)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.invoke(RestFn.java:411)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_one.invokeStatic(core.clj:5961)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_one.invoke(core.clj:5956)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_lib$fn__6875.invoke(core.clj:6003)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_lib.invokeStatic(core.clj:6002)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_lib.doInvoke(core.clj:5981)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.applyTo(RestFn.java:145)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$apply.invokeStatic(core.clj:669)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_libs.invokeStatic(core.clj:6044)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_libs.doInvoke(core.clj:6028)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.applyTo(RestFn.java:140)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$apply.invokeStatic(core.clj:669)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$require.invokeStatic(core.clj:6066)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at cider.nrepl$loading__6814__auto____34744.invoke(nrepl.clj:1)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at cider.nrepl__init.load(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at cider.nrepl__init.<clinit>(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.Class.forName0(Native Method)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.Class.forName(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.classForName(RT.java:2229)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.classForName(RT.java:2238)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.loadClassForName(RT.java:2257)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.load(RT.java:469)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.load(RT.java:444)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load$fn__6933.invoke(core.clj:6189)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load.invokeStatic(core.clj:6188)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load.doInvoke(core.clj:6172)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.invoke(RestFn.java:411)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_one.invokeStatic(core.clj:5961)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_one.invoke(core.clj:5956)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_lib$fn__6875.invoke(core.clj:6003)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_lib.invokeStatic(core.clj:6002)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_lib.doInvoke(core.clj:5981)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.applyTo(RestFn.java:145)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$apply.invokeStatic(core.clj:669)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_libs.invokeStatic(core.clj:6044)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load_libs.doInvoke(core.clj:6028)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.applyTo(RestFn.java:140)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$apply.invokeStatic(core.clj:669)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$require.invokeStatic(core.clj:6066)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at imigresen_api.app.server$loading__6814__auto____32290.invoke(server.clj:1)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at imigresen_api.app.server__init.load(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at imigresen_api.app.server__init.<clinit>(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.Class.forName0(Native Method)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.Class.forName(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.classForName(RT.java:2229)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.classForName(RT.java:2238)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.loadClassForName(RT.java:2257)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.load(RT.java:469)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.load(RT.java:444)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load$fn__6933.invoke(core.clj:6189)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load.invokeStatic(core.clj:6188)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.core$load.doInvoke(core.clj:6172)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RestFn.invoke(RestFn.java:411)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.Var.invoke(Var.java:386)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.Util.loadWithClass(Util.java:251)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at imigresen_api.app.server.<clinit>(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    | Caused by: java.lang.ClassNotFoundException: com.sun.source.doctree.ParamTree
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.Class.forName0(Native Method)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at java.base/java.lang.Class.forName(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.classForName(RT.java:2229)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at clojure.lang.RT.classForName(RT.java:2238)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at orchard.java.parser_next__init.__init0(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   at orchard.java.parser_next__init.<clinit>(Unknown Source)
api_imigresen-api-dev.2.or6liw45kepl@imigresen-dev-instance-qtz6    |   ... 123 more
```

--- a/api/imigresen-api/Dockerfile
+++ b/api/imigresen-api/Dockerfile
@@ -10,7 +10,7 @@ RUN cd skulpture-eventing && lein install
 RUN cd imigresen-common && lein install
 RUN cd imigresen-api && lein build.prod
 
-FROM eclipse-temurin:24-jre-alpine
+FROM eclipse-temurin:24-jdk-alpine
 
 # COPY --from=build /build/imigresen-common/target/common-SNAPSHOT-standalone.jar /app/common-standalone.jar
 COPY --from=build /build/imigresen-api/target/uberjar/api-0.1.0-SNAPSHOT-standalone.jar /app/api-standalone.jar

